search/src/components/Atoms/AfMessage.tsx

```diff
@@ -29,32 +28,7 @@ export const AfMessage = (props: AfMessageProps) => {
   createEffect(() => {
     if (props.streamingCompletion()) return;
     const curOrder = props.order;
-    const bracketRe = /\[(.*?)\]/g;
-    const numRe = /\d+/g;
-    let match;
-    let chunkNums;
-    const chunkNumList = [];
 
-    while ((match = bracketRe.exec(props.content)) !== null) {
-      const chunkIndex = match[0];
-      while ((chunkNums = numRe.exec(chunkIndex)) !== null) {
-        for (const num1 of chunkNums) {
-          const chunkNum = parseInt(num1);
-          chunkNumList.push(chunkNum);
-        }
-      }
-    }
-    chunkNumList.sort((a, b) => a - b);
-    for (const num of chunkNumList) {
-      const chunk = props.chunks()[num - 1];
-      chunk.score = num;
-      if (!metadata().includes(chunk)) {
-        // the linter does not understand that the chunk can sometimes be undefined or null
-        // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
-        if (!chunk) return;
-        setMetadata((prev) => [...prev, chunk]);
```

Comment:
Are we not doing the citations for the chat anymore?

Reply:
We are, but I think it's just better to show all the selected cards all the time. Trying to select only the used ones was too inconsistent.